doc/manual/install-domserver.rst

```diff
@@ -37,7 +37,7 @@ GNU/Linux, or one of its derivative distributions like Ubuntu::
 
   sudo apt install acl zip unzip mariadb-server apache2 \
         php php-fpm php-gd php-cli php-intl php-mbstring php-mysql \
-        php-curl php-json php-xml php-zip composer ntp
+        php-curl php-json php-xml php-zip composer ntp gcc g++
```

Comment:
I think it would be better to here also refer to the judgehost requirements due to #862 than to add the dependencies.